src/Client/ClientBase.cpp

```diff
@@ -2862,7 +2862,10 @@ class OptionsAliasParser
 
 }
 
-
+/// Enable optimizations even in debug builds because otherwise options parsing becomes extremely slow affecting .sh tests
+#if defined(__clang__)
+#pragma clang optimize on
```

Comment:
https://clang.llvm.org/docs/LanguageExtensions.html#extensions-for-selectively-disabling-optimization
Based on this, we cannot enable selectively parts to optimize?

Reply:
Hmm, that's strange, I tested this locally, and it became slightly faster after adding this...

Reply:
Seems like the same binary is generated with and without the tag